Unit and lint

shashwattechsophy opened the pull request:
Test Case coverage upto 95%. added gradle config to check for test coverage up till 85%.

Jagadish-Bisoi committed:
Test Cases

shashwattechsophy committed:
Unit Test Cases

Test Case coverage upto 95%. added gradle config to check for test coverage up till 85%.

VaibhavTechsophy commented on .gradle/buildOutputCleanup/cache.properties:
```diff
@@ -0,0 +1,2 @@
+#Fri Sep 16 12:01:40 IST 2022
+gradle.version=6.1.1
```
these files Should not be committed. Please remove the binary and temp files from commit/merge

VaibhavTechsophy reviewed (changes requested):
Remove all binary and build output files.
Please configure test coverage checks as part of build to 85%.

VaibhavTechsophy reviewed (changes requested):
Test coverage is added, I missed in the list of 98 files.
Proceed to remove the binary files alone.

shashwattechsophy committed:
Revert "Unit Test Cases"

This reverts commit 86a8744dceac46af2f368b1fd7d04e6c7b10df07.

shashwattechsophy committed:
Unit test (Changes based on suggestion)

Test Case coverage upto 95%. added gradle config to check for test coverage up till 85%.

VaibhavTechsophy committed:
Merge branch 'dev' into UnitAndLint
